src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$size: 100px;
+$month-length: 31;
+$sun: 606px;
+
+.calendar {
+  gap: 1px;
```

Comment:
According to the BEM methodology, it's not recommended to use tag selectors or id selectors. You should always use class selectors. This will make your elements more flexible and reusable.